errortrace-doc/errortrace/scribblings/errortrace.scrbl

```diff
@@ -551,6 +565,28 @@ Same as @racket[stacktrace-imports^].}
 
 }
 
+
+@defsignature[stacktrace-filter^ ()]{
+
+The function defined for @racket[stacktrace-filter^] provides additional
+control over the expressions that are instrumented for debugging an
+profiling.
+
+@history[#:added "1.2"]
+
+@defproc[(should-annotate? [stx syntax?] [phase exact-nonnegative-integer?]) any/c]{
+
+Determines whether the @racketout[annotate] function from
+@racket[stacktrace/filter@] or @racket[stacktrace/annotator/filter@]
+adds debugging or profiling annotation to the expression represented
+by @racket[stx]. Annotation is added only when
+@sigelem[stacktrace-filter^ should-annotate?] returns a true value.
+
+The default filter function used by @racket[stacktrace@] and
+@racket[stacktrace/annotator@] annotates an expression if it has a
+source location.}}
```

Comment:
@florence Good points: I made `should-annotate?` apply to test coverage, and I'm not sure whether that was a good idea, so opinions are welcome there. Also, the old filter for profile points was based on `syntax-position` instead of `syntax-source`, and I don't know whether that matters or why it started out different. I didn't make `should-annotate?` filter profile points, because there doesn't seem to be any filter in place currently.